docs/topics/serialization.txt

```diff
@@ -15,7 +15,7 @@ serializer to handle any format (text-based or not).
 Serializing data
 ================
 
-At the highest level, serializing data is a very simple operation::
+At the highest level, are used like this::
```

Comment:
Missing subject. Perhaps: "At the highest level, serializing data can be done like this::"

Reply:
"At the highest level, you can serialize data like this::" to avoid passive voice?

Reply:
Yes good point!